test/pythonml/integration_configuration.yaml

```diff
@@ -68,6 +68,7 @@ unit_shortnames:
 
   # Classifiers
   CLS_randomForest: "pyml:model:random_forest_classification:sklearn.pyi"
+  CLS_gradBoost: "pyml:model:gradboosted_classification:sklearn.pyi"
```

Comment:
Check the naming convention with the other gradient boosting algo above. Technically gradient boosting can be performed with any weak learner, not just a decision tree (although sklearn's implementation only allows for trees).

Reply:
Ah, got it. Will do. Thanks.